helm_deploy/values-dev.yaml

```diff
@@ -21,6 +21,7 @@ generic-service:
     ENVIRONMENT_NAME: DEV
     SENTRY_ENVIRONMENT: DEV
     AUDIT_ENABLED: 'true'
+    FEATURE_ENABLED_PRISONS: 'KMI,GNI,SPI,LGI,DWI,HOI,WWI'
 
```

Comment:
Small one - just wording - FEATURE_ENABLED_PRISONS doesn't say what feature is enabled. ie. it's assuming there is only one feature. Perhaps MANAGE_CONTACTS_ENABLED_PRISONS?